Plugins/Alpakit/Source/Alpakit/Private/AlpakitModEntry.cpp

```diff
@@ -1,93 +1,129 @@
+    return LOCTEXT("PlatformName_Windows", "Windows");
 #elif PLATFORM_MAC
-	return LOCTEXT("PlatformName_Mac", "Mac");
+    return LOCTEXT("PlatformName_Mac", "Mac");
 #elif PLATFORM_LINUX
-	return LOCTEXT("PlatformName_Linux", "Linux");
+    return LOCTEXT("PlatformName_Linux", "Linux");
 #else
-	return LOCTEXT("PlatformName_Other", "Other OS");
+    return LOCTEXT("PlatformName_Other", "Other OS");
 #endif
 }
 
-void SAlpakitModEntry::PackageMod() const {
-	UAlpakitSettings* Settings = UAlpakitSettings::Get();
-	const FString PluginName = Mod->GetName();
-	const FString GamePath = Settings->SatisfactoryGamePath.Path;
-
-	const FString ProjectPath = FPaths::IsProjectFilePathSet() ?
-		FPaths::ConvertRelativePathToFull(FPaths::GetProjectFilePath()) :
-		FPaths::RootDir() / FApp::GetProjectName() / FApp::GetProjectName() + TEXT(".uproject");
-
-	FString AdditionalUATArguments;
-	if (Settings->bCopyModsToGame) {
-		AdditionalUATArguments.Append(TEXT("-CopyToGameDir "));
-	}
-	if (Settings->LaunchGameAfterPacking != EAlpakitStartGameType::NONE) {
-		AdditionalUATArguments.Append(TEXT("-LaunchGame "));
-		AdditionalUATArguments.Append(GetArgumentForLaunchType(Settings->LaunchGameAfterPacking)).Append(TEXT(" "));
-	}
-	
-	const FString LaunchGameArgument = GetArgumentForLaunchType(Settings->LaunchGameAfterPacking);
-
-	const FString CommandLine = FString::Printf(TEXT("-ScriptsForProject=\"%s\" PackagePlugin -Project=\"%s\" -PluginName=\"%s\" -GameDir=\"%s\" %s"),
-	                                            *ProjectPath, *ProjectPath, *PluginName, *Settings->SatisfactoryGamePath.Path, *AdditionalUATArguments);
-
-	const FText PlatformName = GetCurrentPlatformName();
-	IUATHelperModule::Get().CreateUatTask(CommandLine, PlatformName,
-			LOCTEXT("PackageModTaskName", "Packaging Mod"),
+void SAlpakitModEntry::PackageMod(const TArray<TSharedPtr<SAlpakitModEntry>>& NextEntries) const {
+    UAlpakitSettings* Settings = UAlpakitSettings::Get();
+    const FString PluginName = Mod->GetName();
+    const FString GamePath = Settings->SatisfactoryGamePath.Path;
+
+    const FString ProjectPath = FPaths::IsProjectFilePathSet()
+        ? FPaths::ConvertRelativePathToFull(FPaths::GetProjectFilePath())
+        : FPaths::RootDir() / FApp::GetProjectName() / FApp::GetProjectName() + TEXT(".uproject");
+
+    FString AdditionalUATArguments;
+    if (Settings->bCopyModsToGame) {
+        AdditionalUATArguments.Append(TEXT("-CopyToGameDir "));
+    }
+    if (Settings->LaunchGameAfterPacking != EAlpakitStartGameType::NONE && NextEntries.Num() == 0) {
+        AdditionalUATArguments.Append(TEXT("-LaunchGame "));
+        AdditionalUATArguments.Append(GetArgumentForLaunchType(Settings->LaunchGameAfterPacking)).Append(TEXT(" "));
+    }
+
+    const FString LaunchGameArgument = GetArgumentForLaunchType(Settings->LaunchGameAfterPacking);
+
+    UE_LOG(LogTemp, Display, TEXT("Packaging plugin \"%s\". %d remaining"), *PluginName, NextEntries.Num());
```

Comment:
Use LogAlpakit, not LogTemp. LogTemp is not intended to be used outside of local debugging ever.